src/test-run/bookmark.ts

```diff
@@ -158,7 +158,13 @@ export default class TestRunBookmark {
 
             const preserveUrl = this.role.opts.preserveUrl;
 
-            await this._restorePage(this.role.redirectUrl as string, stateSnapshot);
+            if (this.role.redirectUrl) {
```

Comment:
In previous code there was not `null ref` check.
Is the empty redirect url possible now?